_data/navigation.yml

```diff
@@ -73,6 +75,8 @@ docs:
                   url: /cancellations-and-returns/  
                 - title: 'Consignment Tracking'
                   url: /consignment-tracking/
+                - title: 'Context-Driven Merchandising Carousels'
+                  url: /cds-merchandising-carousels/
```

Comment:
Is this still valid? I only see a file for `/cds-integration/` and not for `/cds-merchandising-carousels/`

Reply:
Gaah!!! 😳 
Good catch!! 😅

Reply:
Fixed.